tests/GrabyFunctionalTest.php

```diff
@@ -312,14 +328,17 @@ public function testMultipage(): void
 
     public function testCookie(): void
     {
+        $httpMockClient = new HttpMockClient();
+        $httpMockClient->addResponse(new Response(301, ['Server' => ['AkamaiGHost'], 'Content-Length' => ['0'], 'Location' => ['https://www.npr.org/sections/parallels/2017/05/19/529148729/michael-flynns-contradictory-line-on-russia'], 'Date' => ['Sat, 08 Jan 2022 05:02:29 GMT'], 'Connection' => ['keep-alive']], (string) file_get_contents(__DIR__ . '/fixtures/content/http___www.npr.org_sections_parallels_2017_05_19_529148729_michael-flynns-contradictory-line-on-russia.html')));
+        $httpMockClient->addResponse(new Response(200, ['Content-Type' => ['text/html;charset=UTF-8'], 'X-Cache-NPR' => ['MISS'], 'Access-Control-Allow-Origin' => ['http://www.google.co.uk'], 'Access-Control-Allow-Credentials' => ['true'], 'X-NPR-Trace-ID' => ['renIaLEEV1Q'], 'X-Content-Type-Options' => ['nosniff'], 'X-XSS-Protection' => ['1; mode=block'], 'X-Served-By' => ['pod-www-render-nginx-776fbf4c8c-7sjbx'], 'Referrer-Policy' => ['no-referrer-when-downgrade'], 'Strict-Transport-Security' => ['max-age=15724800; includeSubDomains'], 'Cache-Control' => ['no-cache'], 'Expires' => ['Sat, 08 Jan 2022 05:02:30 GMT'], 'Date' => ['Sat, 08 Jan 2022 05:02:30 GMT'], 'Transfer-Encoding' => ['chunked'], 'Connection' => ['keep-alive', 'Transfer-Encoding']], (string) file_get_contents(__DIR__ . '/fixtures/content/http___www.npr.org_sections_parallels_2017_05_19_529148729_michael-flynns-contradictory-line-on-russia')));
```

Comment:
I guess this will not actually do anything if we want to send cookie.

Reply:
You're right, could you add an exclusion rule for Rector to not auto mock that request?

Reply:
There are three options for this:

- Have blacklist of function names
- Use some in code marker (e.g. a comment like `Rector: do not add mocking`
- Pass `null` as second argument (with a comment for humans that we do not want mocking) – this one works now.

What would you prefer.

Reply:
The code marker is an annotation to the function to blacklist?

Reply:
I'm ok with that

Reply:
It is now skipped.